### ISSUE TITLE
Consider taking no-multiline-string out of the recommended preset

### ISSUE THREAD

**Issue by astorije** [Microsoft/tslint-microsoft-contrib#454]
We use and love the recommended preset at @CloudHealth, but we find `no-multiline-string` to be an unnecessarily restrictive rule to recommend by default.

As far as I can tell, there is nothing inherently bad about multiline template strings, and they're not one of those things accepted to be dangerous or recommended not to use by the community such as `eval` or `==`.

In our case, and this is probably the biggest reason why we disabled it (but not the only one), GraphQL queries are most of the time represented using multiline template strings:

```graphql
query UserQuery($email: string{
  user(id: $email) {
    name
    avatar
  }
}
```

This is true with both Apollo Client and Relay Client.

We also find this useful when building CLIs for example, but it's less easily found across our codebase.

So there is it, I suggest removing it from the recommended preset, from the point of view of a consumer :)
Thanks and keep up the good work!

**Comment by JoshuaKGoldberg** [comment]
This sounds reasonable (I too always disable it 😃). Let's wait a bit and see if there are any voices against it, and if in a few weeks time nobody has piped up, it sounds like a good `Good First Issue 🙌`.

**Comment by astorije** [comment]
> Let's wait a bit and see if there are any voices against it

That's totally reasonable. I opened #456 in preparation for it, so that when/if no one objects, it can be merged, or close if someone objects.
Thanks for the feedback @JoshuaKGoldberg!
